src/App.tsx

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
-import './App.css';
+import Header from './components/Header';
+import Aside from './components/Aside';
+import CalendarMain from './components/CalendarMain';
+import { useState } from 'react';
 
 function App() {
+  const [currentDate, setCurrentDate] = useState(new Date());
+  const handleDateChange = (newDate: Date) => {
+    setCurrentDate(newDate);
+  };
   return (
-    <div className="App">
-        <h1>ok</h1>
+    <div className="app-container">
+      <Header onDateChange={handleDateChange} date={currentDate}></Header>
+      <Aside date={currentDate}></Aside>
+      <CalendarMain date={currentDate}></CalendarMain>
```

Comment:
Naming could be improved, let't try to avoid generic words.
```suggestion
      <WeekCalendar date={currentDate} />
```
Also if there are no children you can use shorter syntax.